obs_status.yml

```diff
@@ -0,0 +1,2147 @@
+bad:
+  5115672: 9
+  33690776: 9
+  34996472: 9
+  38535536: 9
+  40113272: 9
+  45616472: 9
+  47846640: 9
+  48892720: 9
+  49818760: 9
+  62922104: 9
+  68956456: 9
+  92812824: 9
+  107881800: 9
+  109061768: 9
+  110493968: 9
+  111028544: 9
+  111807640: 9
+  118359864: 9
+  118498608: 9
+  120457376: 9
+  144714600: 9
+  145113736: 9
+  152831696: 9
+  161357600: 9
+  166993856: 9
+  185871616: 9
+  187829568: 9
+  188482256: 9
+  188501784: 9
+  188623712: 9
+  188750920: 9
+  195167088: 9
+  225576592: 9
+  239080480: 9
+  257953336: 9
+  264115840: 9
+  288753656: 9
+  306200176: 9
+  311824376: 9
+  327945264: 9
+  328077952: 9
+  330967424: 9
+  350392600: 11
+  373564272: 9
+  377102496: 9
+  391120072: 9
+  396501520: 9
+  405276920: 9
+  417611952: 11
+  444743456: 11
+  450103168: 9
+  458500128: 13  # new value
```

Comment:
Where is this new value defined?

Reply:
It has not been defined. This was not added because of the magnitude being an upper/lower limit, which are the values known up to now. This star would need your review.

Reply:
OK, so you wanted to add this to the bad star list because it has variability not captured in the agasc and didn't have a mag code for that.

So should we just add a mag entry with mean mag and an error that captures the variability?  I don't see that it would even really need to be on the bad table but right now the bad table seems to be the only place we have to document our decisions about the values in the manual mags table?

Reply:
I also note that, somewhat similar to the variability seen in star 1105998760 in 48040, the variability here in 458500128 in obsid 17085 seems to be related to the dither pattern.

Reply:
I've removed this star from the bad list as it looks like the behavior was related to an area on the ccd / camera and not the star.